edx_rest_framework_extensions/middleware.py

```diff
@@ -72,6 +73,13 @@ def _set_all_request_attributes(self, request):
         """
         Sets all the request custom attributes
         """
+        # .. custom_attribute_name: edx_drf_extensions_version
+        # .. custom_attribute_description: The version of the edx-drf-extensions library installed, which may be
+        #   useful when trying to rollout important changes to all services. Note that RequestCustomAttributesMiddleware
+        #   must be installed for this to work. Also, versions before 8.7.0 will not include this attribute, but
+        #   should have ``request_auth_type_guess``.
```

Comment:
What's the importance of this line? Is it communicating a way of determining that the library is installed, but at an older version?